### ISSUE TITLE
When connecting to user on the same machine, connection just closes when user logs in

### ISSUE THREAD

**Issue by IcyTower1337** [stascorp/rdpwrap#1036]
I'm trying to connect to a user on the same machine, and I'm able to start the connection and insert the password for the user, but as soon as it logs in, the connection just closes instantly. Not sure what's wrong and how to fix it.

My RDPConfi.exe: (I got the updated .ini)
![image](https://user-images.githubusercontent.com/25158289/77677710-f9c61d80-6f87-11ea-9a42-801d381f24d7.png)

Video of it happening:
https://puu.sh/FpfD4/63314cb130.mp4

Any idea what I'm doing wrong?

**Comment by affinityv** [comment]
Maybe the user takes his session back?

**Comment by IcyTower1337** [comment]
Not sure how that would happen as the user isn't the same I'm using, I'm trying to rdp into another user account on the same machine, and its session isn't active, and when I try using RDPCheck.exe, the only user that actually exists is my normal user not the one I'm trying to RDP into.
![image](https://user-images.githubusercontent.com/25158289/77685246-187de180-6f93-11ea-9338-bb3b51e5c5b7.png)

Also can't write the name of the user on the Other User tab to login, it doesn't seem to detect it.

**Comment by affinityv** [comment]
Okay, it may be the authentication choice being no good for the other machine?  Or perhaps the other user isn't in RDP group.... so, not allowed to use RDP (I would expect a reasonable error message for both though).

**Comment by IcyTower1337** [comment]
It's the same machine though, So I don't see how it wouldn't be able to detect something that is local?
I did think it was a per user config problem, but when I logged into the `RDP-AFKUser` and checked the RDPConf.exe, it seemed fine, and I checked the remote settings, and it was configured to allow remote access to it, aswell as being listed on the advanced group settings on remote. So I've got no clue, does this program not allow RDP'ing into the same machine but different user? Or is it somehow misconfigured on my end?

**Comment by affinityv** [comment]
Is that other account somehow disabled or hidden?  It should show on the list of users with RDPCheck...

**Comment by IcyTower1337** [comment]
I don't think so, because I can literally sign into it if I just switch users... it just isn't there if I use RDPCheck.

**Comment by IcyTower1337** [comment]
Just so I can be sure, do you know if it's possible to do what I'm trying to achieve? As in use Remote Desktop Connection to sign in to another local user? Because I might just be trying to do something that isn't possible with or without this...

**Comment by affinityv** [comment]
Okay, I often remote to 127.0.0.2 as the host on the remote system when I am already on the remote system.

**Comment by IcyTower1337** [comment]
Okay, so I at least know the issue is in my end, and I just gotta try and mess around until I find what is wrong. 

If you don't mind me asking a couple more questions since you can remote to your own machine but a different user, you can have both users (the host which you use and the other one you RDP into) open at the same time right?

Just trying to see if it's worth the probably hours of messing about trying to fix before I spend too much time on it.

**Comment by affinityv** [comment]
Yes, I commonly connect to one machine via another; and with RDP Wrapper, sometimes back to the same machine instead of on to another.
